README.md

```
@@ -3,0 +1,73 @@
+
+## Table of Contents
+
+- [Build Configuration](#build-configuration)
+- [Images](#images)
+  - [Ubuntu](#ubuntu)
+  - [PHP](#php)
+  - [Scala](#scala)
+  - [Jenkins](#jenkins)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)